spring-cloud-azure-feature-management-web/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The indention for this pom file is different with other existing pom files, e.g., [context pom file](https://github.com/microsoft/spring-cloud-azure/blob/master/spring-cloud-azure-context/pom.xml). Better to align with other existing pom files in the same project. Another pom file as well.